designer/client/src/http/HttpService.ts

```diff
@@ -123,12 +121,23 @@ class HttpService {
         this.#notificationActions = na;
     }
 
-    loadBackendNotifications(): Promise<BackendNotification[]> {
+    loadBackendNotifications(
+        onSuccess: () => void,
+        onError: (error: any, defaultErrorHandler: () => void) => void,
+    ): Promise<BackendNotification[]> {
         return api
             .get<BackendNotification[]>("/notifications")
-            .then((d) => d.data)
+            .then((d) => {
+                onSuccess();
+                return d.data;
+            })
             .catch((error) => {
-                this.#addError(i18next.t("notification.error.cannotFetchBackendNotifications", "Cannot fetch backend notification"), error);
+                onError(error, () => {
+                    this.#addError(
```

Comment:
Hmm, do we need this `#` before method name?

Reply:
I didn't create this error, but I assume, someone wants to make this variable private https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields so i would say, yes :)

Reply:
Ehh java script.. :) Let's leave it.